model/edgeless.ml

```diff
@@ -118,6 +124,7 @@ module Orchestrator : sig
 
   val start : t -> wid * Function.t -> t * (nid * fid) list
   val stop : t -> nid * fid -> t
+  val stop : t -> wid -> t
```

Comment:
Can't overload values -- suggest `delete_workflow` rather than the second `stop` perhaps? (Or just `delete` if that's clear enough?)